Doc/library/sqlite3.rst

```diff
@@ -2440,9 +2440,13 @@ or if :attr:`~Connection.autocommit` is ``True``,
 the context manager does nothing.
 
 .. note::
-
    The context manager neither implicitly opens a new transaction
-   nor closes the connection.
+   nor closes the connection. Using ``contextlib.closing``, the context manager
+   can be used for connection management. In this case, the ``sqlite3``
+   connection is closed without an additional ``sqlite.connect.close()`` after a
+   context manager closes. For default ``sqlite`` context management cases which don't use
+   ``contextlib.closing``, this is intended to perform multiple transactions without3fully
+   closing the connection.
```

Comment:
Hm, I still think this is too verbose. Try to keep it too the point. The existing sentence ends with _"nor closes the connection"_. Try using that to spin off the following sentence. How about something like this:

```suggestion
   nor closes the connection.
   If you need a closing context manager,
   consider using :meth:`contextlib.closing`.
```

@ezio-melotti, do you think "a closing context manager" can be implicitly understood as "a context manager that closes the conntection"? I wanted to avoid repeating the _"closes the connection"_ part.

Reply:
Mr @erlend-aasland , am requesting for a review of my work, thanks